FlagsmithClient/Classes/Flagsmith.swift

```diff
@@ -215,4 +258,41 @@ public class Flagsmith {
       completion(result)
     }
   }
+  
+  /// Return a flag for a flag ID and identity, using either the cache (if enabled) or the default flags
+  private func getFlagUsingDefaults(withID id: String, forIdentity identity: String? = nil) -> Flag? {
+    return self.defaultFlags.first(where: {$0.feature.name == id})
+  }
+
+  /// Return an array of flags for an identity, including the cached flags (if enabled) and the default flags when they are not already present in the passed array
+  private func getFlagsUsingDefaults(flags:[Flag], forIdentity identity: String? = nil) -> [Flag] {
```

Comment:
2 comments on this function: 

 1. I don't think the `flags` argument is ever passed as anything other than `[]` so we can probably remove it? 
 2. The comment above mentions that it uses the cache, but I don't think that's true (probably not since we moved to `URLCache`).

Reply:
Based on the above, perhaps we can remove this entirely and just use `self.defaultFlags` wherever we were previously using this function?

Reply:
@matthewelwell fair point, I've tidied that up as well now.